### PR TITLE
Add `loss_agg_mode` for kl and entropy_loss

### PR DESCRIPTION
**Pull request by pan-x-c**
## Description

Align with verl 0.5.0

1. Add `loss_agg_mode` for kl and entropy_loss
2. Update `PPOPolicyLoss`
3. Add 'low_var_kl'
4. Update `masked_sum`

## Checklist

Please check the following items before code is ready to be reviewed.

- [ ]  Code has passed all tests
- [ ]  Docstrings have been added/updated in Google Style
- [ ]  Documentation has been updated
- [ ]  Code is ready for review

### CODE REVIEW

File: trinity/trainer/verl/dp_actor.py
Comment (hiyuchang) [inline trinity/trainer/verl/dp_actor.py:187]: Suggest adding `micro_batch_metrics.update({"actor/final_loss": loss.detach().item()})` before `loss.backward()` to indicate the adopted loss (including kl and entropy if any). Ref: https://github.com/volcengine/verl/blob/main/recipe/sppo/dp_actor.py

Or, compute and add mini-batch level loss`"actor/final_loss": total_loss / len(micro_batches)`
